extension/assets/js/options.js

```diff
@@ -22,11 +22,23 @@ $(document).ready(function() {
   $("#main").hide();
 
   HFX.Settings.getAll((items) => {
-    for (const section in sections) {
+    const sortedSections = Object.keys(sections).sort();
+
+    for (const section of sortedSections) {
       addSectionToList(section);
       buildSectionBase(section);
 
-      for (const feature in sections[section]) {
```

Comment:
Didn't this already sort features by the property in order?

Reply:
I definitely noticed sections getting reordered. Also, for the Alerts module, the name is `HFX Alerts` but the class is `Alerts` so it looks strange to sort by class rather than name.

The order here would ultimately depend on how `browserify` bundles the modules because the order in the section object depends on order of instantiation.